ringpop.go

```diff
@@ -96,6 +96,25 @@ func mergeDefault(opts *Options) *Options {
 	return opts
 }
 
+type Interface interface {
```

Comment:
Calling this `Interface` doesn't sit right with me. Mocks generated with `mockery` will give it an ambiguous name:

```
$ mockery -name Interface -output testutils/mocks/
Generating mock for: Interface
```

Also referring to the Ringpop interface within the package will mean signatures like:
```
func foo(r Interface)
```

I haven't come across any convention that sits perfectly with me, but I'd probably prefer `ringpop.RingpopInterface` over `ringpop.Interface`. Especially since third-party packages may import the interface.